legacy/ui/legacy/src/main/java/com/fsck/k9/ui/messageview/MessageTopView.kt

```diff
@@ -80,6 +84,9 @@ class MessageTopView(
 
         containerView = findViewById(R.id.message_container)
 
+        renderPlainFormat =
+            (generalSettingsManager.getConfig().display.visualSettings.bodyContentType == BodyContentType.TEXT_PLAIN)
+
```

Comment:
In case you choose to go with the injection of `DisplayVisualSettingsPreferenceManager` instead, you could simplify to:

```suggestion
        renderPlainFormat =
            visualSettingsPrefManager.getConfig().bodyContentType == BodyContentType.TEXT_PLAIN
```

Reply:
Updated.